deploy/1-create-operator.yaml

```diff
@@ -633,24 +635,16 @@ spec:
                   - rbac
                   type: string
                 analyticsConfigAuditTTL:
-                  description: Time (in hours) the analytics config data entering the
```

Comment:
Please consider cherry-picking to your branch this commit https://github.com/Juniper/contrail-operator/pull/363/commits/8af3424f09d6207069a66272960b327b43313eb6 so in your change you will not remove analytics fields descriptions. I fixed that in master branch, did not know that 2008 branch is still active.

Reply:
Will take care of it, Thank you for notifying.